### PR TITLE
New post on walls

### PR DESCRIPTION
**Pull request by rbbri**
💵

### CODE REVIEW

**Comment by rbbri** [comment]
fixes #79

File: app/models/user.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 class User < ApplicationRecord
   has_many :posts
   has_many :comments
+  accepts_nested_attributes_for :posts
```
Comment (Caitlin-cooling) [inline app/models/user.rb]: What is this actually doing?

**Comment by Caitlin-cooling** [comment]
:shipit:

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -9,3 +9,23 @@
     </section>
   </aside>
 </div>
+<div>
+  <%= form_for :post, url: "/#{@user.id}" do |form| %>
+  <%= form.label :message %>
+  <%= form.text_area :message %>
+    <%= form.fields_for :users do |user| %>
```
Comment (Caitlin-cooling) [inline app/views/users/show.html.erb]: Do we need this?

Comment (rbbri) [reply app/views/users/show.html.erb]: No

File: app/models/user.rb (hunk 1 above)
Comment (rbbri) [reply app/models/user.rb]: Nothing!

**Pull request merged**